internal/view/app.go

```diff
@@ -173,6 +172,49 @@ func (a *App) suggestCommand() model.SuggestionFunc {
 	}
 }
 
+func (a *App) suggestSubCommand(command string) []string {
+	cmds := strings.Split(command, " ")
+	if len(cmds[0]) == 0 || len(cmds) != 2 {
+		return nil
+	}
+
+	var suggests []string
+	switch strings.ToLower(cmds[0]) {
+	case "cow", "q", "q!", "qa", "Q", "quit", "?", "h", "help", "a", "alias", "x", "xray", "dir":
+		return nil // ignore special commands
+	case "ctx", "context", "contexts":
+		ctxs, err := a.factory.Client().Config().Contexts()
+		if err != nil {
+			log.Error().Err(err).Msg("failed to list contexts")
+			return nil
+		}
+
+		for contextName := range ctxs {
+			if suggest, ok := shouldAddSuggest(cmds[1], contextName); ok {
```

Comment:
Also it would be great to add fuzzy matching here (we do this with filter commands) so user could type `po ed` and suggest `fred, bleed`

Reply:
Using fuzzy matching can result in very strange views. For example, there are these namespaces: `kube-system, kube-storage`, if I type: `po ks` it comes up with `po ks[gray]kube-system[gray]`, or if I type `po age` it comes up with `po age[gray]kube-storage[gray]`. , I think this may be not understandable for users, what do you think?

Reply:
Right! This could be a double edge sword indeed. 
Perhaps using fuzzy logic and returning the top 3 scored suggestions could make sense? 
I was looking at #2163 so perhaps checking for contains vs prefix would help in this case? but could also yield additional user headaches?

Perhaps outside the scope of this PR and we could experiment with this a bit and see how we could address this better in the future.

If you have other ideas, I am all ears!